### PR TITLE
[FIX] account: fix payment view

### PR DESCRIPTION
**Pull request by vin-odoo**
The payment view has a o_attachment_preview div inside it. From the look of it, this is a remnant of the past, as the preview is added next to the form view automatically and does not need to be included inside it.
What happens at the moment is that the div appears in some case either empty but taking some place, or even duplicating the attachment.

Task id #3104741

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by h4818 — APPROVED** [review]

**Comment by h4818** [comment]
What about 16.0?

**Comment by vin-odoo** [comment]
@h4818 Indeed, he issue is there too. I tried retargeting this pr to 16.0, it seems ok from the look of the dashboard :)

File: addons/account/views/account_payment_view.xml [hunk 1]
```diff
@@ -312,7 +312,6 @@
                             </group>
                         </group>
                     </sheet>
-                    <div class="o_attachment_preview"/>
```
Comment (smetl) [inline addons/account/views/account_payment_view.xml:315]: Are you removing a feature in a stable version? :laughing: 
on 16.0:
image

on your branch:
image

**Review by smetl — CHANGES_REQUESTED** [review]

File: addons/account/views/account_payment_view.xml (hunk 1 above)
Comment (vin-odoo) [reply addons/account/views/account_payment_view.xml:315]: Hmm I'll check again, as you can see I retargeted from master and was expecting the same behaviour.
In master, I had the preview there even without this node, as another code added it.
(And was able to have the preview showing twice)

**Comment by smetl** [comment]
Will be handled by the js team:
https://github.com/odoo/odoo/compare/16.0...odoo-dev:odoo:16.0-fix-dialog-attachment-preview-mcm

**Pull request closed without merging**